chart/templates/cron-jobs/metrics-collector/_container.tpl

```diff
@@ -5,14 +5,12 @@
 - name: "{{ include "name" . }}-metrics-collector"
   image: {{ include "jobs.cacheMaintenance.image" . }}
   imagePullPolicy: {{ .Values.images.pullPolicy }}
-  env:
-  {{ include "envLog" . | nindent 2 }}
-  {{ include "envQueue" . | nindent 2 }}
-  {{ include "envCommon" . | nindent 2 }}
-  {{ include "envMetrics" . | nindent 2 }}
-  - name: CACHE_MAINTENANCE_ACTION
```

Comment:
Could we keep this variable? It is needed as input of cron job

Reply:
done